articles/you-are-not-a-software-engineer.md

```diff
@@ -0,0 +1,58 @@
+---
+authorName: Stefan Djelekar
+authorGithubUsername: djelekar
+title: "You are not a Software Engineer" - What am I then?
+---
+# {{page.title}}
+
+The job of Software Engineer, or any technical role for that matter, is vast. You might spend more time than you would like on meetings, you might spend your afternoon sending emails to your stakeholders, you actually might only get to write code for a couple of hours a day. Depending on your personality type, this can make you thrilled or make you unhappy. I am writing this for the latter.
+
+This article is not about roles. Nor about daily duties. In this article, I offer a different view, in which our job is solving problems of business and making our customers happy, while coding is only a marginal tool in doing so. I maybe hit a nerve on this one, but it is even more important when working as a service provider. So hear me up.
+
+## "But I am a coder, I don't want to deal with any of this."
+
+Yes, I understand and I am too. Coding is fun, coding can be challenging, coding is the reason most of embarked on this journey. But if you think about it, code is made for the human consumption first. Yes, machines need to understand it as well, but if it wasn't for the people, we would still write in assembly, or binaries even. When you throw in daily collaboration with your team, managers, and clients, your job is really about dealing with people, not computers.
```

Comment:
Missing word: "... most of _us_ embarked..."

Reply:
Fixed.